bin/riemann-elasticsearch

```diff
@@ -36,7 +37,7 @@ class Riemann::Tools::Elasticsearch
   end
 
   def health_url
-    "http://#{options[:es_host]}:#{options[:es_port]}/_cluster/health"
+    "http://#{options[:es_host]}:#{options[:es_port]}/#{options[:path_prefix]}/_cluster/health"
```

Comment:
Is a double slash (//) gonna work? Looks like the default is an empty string, so I want to make sure we don't break existing users.